### PR TITLE
feat: Save output to file

### PR DESCRIPTION
**Pull request by AlirezaKhadem**

### CODE REVIEW

**Comment by AlirezaKhadem** [comment]
related to this feature request: https://github.com/TheYahya/enola/issues/27

File: cmd/enola/find.go [hunk 1]
```diff
@@ -43,3 +54,62 @@ func findAndShowResult(username, site string) {
...
+
+func getOutputfile(outputPath string, filename string) (*os.File, error) {
+	fullPath, err := filepath.Abs(outputPath)
+	if err != nil {
+		fmt.Println("Error resolving absolute path:", err)
+		return nil, err
+	}
+
+	if _, err := os.Stat(filepath.Join(fullPath, filename)); os.IsNotExist(err) {
+		file, err := os.Create(filepath.Join(fullPath, filename))
```
Comment (TheYahya) [inline cmd/enola/find.go:85]: I think it's better if we don't left any err unchecked.

File: cmd/enola/find.go [hunk 2]
```diff
@@ -43,3 +54,62 @@ func findAndShowResult(username, site string) {
 		os.Exit(1)
 	}
 }
+
+func writeOutput(m *model, outputPath string, filename string, printFound bool) error {
```
Comment (TheYahya) [inline cmd/enola/find.go:58]: Just a nitpicking, But I would prefer to have these file related functions in a separate file.

**Comment by TheYahya** [comment]
Hey @AlirezaKhadem, Nice work. Thanks for the PR. 

I would prefer to have the `-o` flag to be exactly name of the output file. Currently if I run `enola theyahya -o output.txt` it will generate a `output.txt` directory with a `theyahya` file inside it. I would prefer to just have `output.txt` file with the results.

**Comment by TheYahya** [comment]
Hi @AlirezaKhadem Jan, 
Closing, since you haven't fixed the issues, and #32 is already merged.

**Pull request closed without merging**